hdt-lib/src/hdt/BasicHDT.cpp

```diff
@@ -656,7 +657,7 @@ void BasicHDT::loadFromHDT(std::istream & input, ProgressListener *listener)
 	controlInformation.load(input);
 	std::string hdtFormat = controlInformation.getFormat();
 	if(hdtFormat!=HDTVocabulary::HDT_CONTAINER) {
```

Comment:
But, as per [my above comment](https://github.com/rdfhdt/hdt-cpp/pull/36/files#r69955509), this check would need to change then.